packages/blade-old/src/icons/PayoutLink/PayoutLink.native.js

```diff
@@ -0,0 +1,24 @@
+import * as React from 'react';
+import PropTypes from 'prop-types';
+import Svg, { Path } from 'react-native-svg';
+import { getThemeColors } from '../../_helpers/theme';
+import colors from '../../tokens/colors';
+
+export default function PayoutLink({ width = 24, height = 24, fill = colors.sapphire[800] }) {
```

Comment:
why do we need to add default color?

Reply:
It is just a fallback case, fill color is not provided it will be filled by this shapphire[800].

All the existing icon is also using this as a default color.